CONTRIBUTING.md

```diff
@@ -138,29 +138,22 @@ You can download Eclipse and load the Frank!Framework sources into it using the
 
 ### Install Eclipse with Lombok
 
-- Download Eclipse from [here](https://www.eclipse.org/downloads/packages/), choosing "Eclipse IDE for Enterprise Java Developers". We tested our instructions with version 2021-09, but older versions should also work. Note that 64-bit Eclipse doesn't work with 32-bit JRE/JDK (doesn't start without any message). There is no installer. To install Eclipse, just unzip your download to a directory of your choice.
+- Open the webpage with [downloads of Eclipse](https://www.eclipse.org/downloads/packages/). At the top of this page, you see a link to download an installer. We recommend that you do not use an installer, because you can also download a .zip file with the sources of Eclipse. When you use a zip file instead of an installer, it is easier to have different versions of Eclipse on your development computer.
+- Click the link "Eclipse IDE for Enterprise Java and Web Developers". A page opens with a big Download button to the right. Skip that one because it is an installer. Click a link to the left of that, under "Download Links". We tested our instructions with version 2021-12, but older versions should also work. Note that 64-bit Eclipse doesn't work with 32-bit JRE/JDK (doesn't start without any message). To install Eclipse, just unzip your download to a directory of your choice.
```

Comment:
Gebruikt Eclipse niet z'n eigen JDK? Doet hij het wel met een 64 bit JDK?

Reply:
Ik heb de zin over 64 bit en 32 bit weggehaald. Jaren geleden kon je 64 bit en 32 bit nog door elkaar halen, maar ik denk dat nu iedereen gewoon 64 bit programma's draait.

Later in de tekst raad ik aan om de Zulu OpenJDK te gebruiken. Ik weet dat de unit testen van de Frank!Doc niet kunnen draaien op een JRE, maar alleen op een JDK. Jaco heeft in het kader van de Frank!Runner ervaring met de Zulu OpenJDK.